unlock-app/src/components/interface/svg/Save.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
```

Comment:
Same here.

Reply:
Same here regarding making the Save button a separate PR?

Or do you mean to merge components/interface/svg/Save.js with buttons/lock/Save?